libs/ballot-interpreter-nh/test/fixtures/amherst-2022-07-12/election.json

```diff
@@ -1,6 +1,6 @@
 {
   "ballotLayout": {
-    "paperSize": "legal",
+    "paperSize": "letter",
```

Comment:
I'm a little confused as to why this became `letter`. Assuming there's a clear explanation and I'm just not seeing it, approved :)

Reply:
Because the actual PDF is letter size but the XML says legal size, and we were incorrectly using legal size. The PR fixes that issue and updates this fixture.